Documentation/Configuration/ConfigurationFiles.rst

```diff
@@ -0,0 +1,26 @@
+.. include:: /Includes.rst.txt
+
+.. _configuration-files:
+
+===================
+Configuration files
+===================
+
+The configuration files :file:`LocalConfiguration.php` and
+:file:`AdditionalConfiguration.php` are located in the directory
+:ref:`public/typo3conf/ <directory-public-typo3conf>` in Composer-based
```

Comment:
The reference is not available by now. It is one that is defined in a following PR?
Below the typo3conf ref also.

Reply:
Yes they are in the follow up. Shall we leave them in?

Reply:
It would be cleaner. But I am fine with both.

Reply:
done